Documentation/specs/runtime-configuration-file.md

```diff
@@ -117,8 +122,8 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 
 This section is copied verbatim from an identical section in the input `project.json` file (with the exception of the `target` parameter which is generated by the compilation process). The `runtimeConfig` section specifies parameters to be provided to the runtime during initialization. Known parameters include:
 
-* `gcServer` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
-* `gcConcurrent` - Boolean indicating if background garbage collection should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).
+* `System.GC.Server` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
+* `System.GC.Concurrent` - Boolean indicating if background garbage collection should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).
```

Comment:
> Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).

This isn't really true anymore, at least not exactly. Originally the name I used was designed to match the `app.config` setting exactly. I do like the new name, but maybe rephrase to something like:

> This is equivalent to the `gcConcurrent` setting in the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx)

(Same for GC Server above, and the other new settings if they have equivalents).

Having the link between old and new settings will be useful for documenting later